vllm/config.py

```diff
@@ -138,12 +138,10 @@ def __init__(
         self.quantization = quantization
         self.quantization_param_path = quantization_param_path
         self.enforce_eager = enforce_eager
-        self.max_context_len_to_capture = max_context_len_to_capture
-        if self.max_context_len_to_capture is not None:
+        if max_context_len_to_capture is not None:
```

Comment:
@WoosukKwon this has been removed over 6 months ago, do you think we still need to maintain this?